app/views/admin/consumer_apps/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="crayons-card p-3 mb-5 mt-2">
+  <%= link_to "New Consumer App", new_admin_consumer_app_path, class: "crayons-btn float-right" %>
+
+  <h1 class="crayons-title mb-6">Consumer Apps</h1>
+
+  <p>
+    The following are apps that will be able to receive Push Notifications from <%= SiteConfig.community_name %>.
+  </p>
+</div>
+
+<table class="crayons-table" width="100%">
+  <thead>
+    <tr>
+      <th scope="col">App Bundle</th>
+      <th scope="col">Platform</th>
+      <th scope="col">Devices</th>
+      <th scope="col">Authentication Key</th>
```

Comment:
why show the authentication key in the index view? By removing it we save quite a bit of real estate leaving only the important info

Reply:
I was considering it to be visual feedback as well. Creators can tell if the Authentication Key is available or not and that helps diagnose what the problem is (why their Consumer Apps aren't working).

Self-hosted Forems will not have an authentication key for the official Forem apps (because it's not available in their ENV variables). That can help them know that "the official Forem apps aren't operational because they don't have an authentication key", and then go into the docs (soon 🙂 ) and read more about it.

Reply:
Ah got it! Maybe we can replace it with a checkmark as well?